src/main/java/uk/gov/justice/digital/delius/controller/secure/AuthenticationController.java

```diff
@@ -65,6 +67,18 @@ public UserDetails findUser(@ApiParam(name = "username", value = "LDAP username"
                 .orElseThrow(() -> new NotFoundException(String.format("User with username %s", username)));
     }
 
+    @ApiOperation(
+            value = "Find user details of a user held in Delius Identity (LDAP)")
+    @ApiResponses(
+            value = {
+                    @ApiResponse(code = 400, message = "Invalid request", response = ErrorResponse.class),
+                    @ApiResponse(code = 401, message = "Unauthorised", response = ErrorResponse.class),
+            })
+    @RequestMapping(value = "/users/search/email/{email}/details", method = RequestMethod.GET)
```

Comment:
maybe `search` path is redundant

Reply:
there was a reason i included this, but i'm happy to take it out.

the reason was to differentiate between the 'get user' style endpoints, which return either a user or 404, and a 'search' style endpoint, which returns a list of users (empty if not found). i figured perhaps in the future more 'search' style endpoints may be added for e.g. searching for a user using just part of the user name or a regex or something. e.g. `/users/search/{username}/details` which returns the same list style results.